### PR TITLE
Project workspace import

### PR DESCRIPTION
**Pull request by abhilash-kumar-nair**

### CODE REVIEW

**Comment by abhilash-kumar-nair** [comment]
PR contains improvements and adds support for new NDM endpoints Ping @efredriksson-modelon , @iakovn , @anna-lindholm :)

File: modelon/impact/client/client.py [hunk 1]
```diff
@@ -420,3 +446,25 @@ def get_project_matchings(
...
+
+        Returns:
+
+            ProjectImportOperation --
+                An modelon.impact.client.operations.project_import.
+                ProjectImportOperation class object.
+
+        Example::
+
+            client.import_project_from_zip(path_to_project)
```
Comment (efredriksson-modelon) [inline modelon/impact/client/client.py:467]: Same: project = X.wait()

File: modelon/impact/client/entities/workspace.py [hunk 2]
```diff
@@ -668,3 +669,51 @@ def get_shared_definition(self, strict: bool = False):
...
+
+        Returns:
+
+            ProjectImportOperation --
+                An modelon.impact.client.operations.project_import.
+                ProjectImportOperation class object.
+
+        Example::
+
+            workspace.import_dependency_from_zip(path_to_project)
```
Comment (efredriksson-modelon) [inline modelon/impact/client/entities/workspace.py:714]: Same; project = X.wait()

File: tests/impact/client/operations/test_external_result.py [hunk 3]
```diff
@@ -1,70 +0,0 @@
-import pytest
```
Comment (efredriksson-modelon) [inline tests/impact/client/operations/test_external_result.py:1]: Was this file meant to be removed without adding any corresponding tests for the operation?

Comment (abhilash-kumar-nair) [reply tests/impact/client/operations/test_external_result.py:1]: Added the tests back :)

**Comment by abhilash-kumar-nair** [comment]
@efredriksson-modelon, Thanks for the review! I have made the suggested fixes. :)

**Review by efredriksson-modelon — APPROVED** [review]
:+1:

**Pull request merged**